src/BucketTesting/Bucket.php

```diff
@@ -22,7 +22,7 @@ public function getName(): string {
 	}
 
 	public function getId(): string {
-		return $this->getCampaign()->getName() . '.' . $this->getName();
+		return 'campaigns.' . $this->getCampaign()->getName() . '.' . $this->getName();
```

Comment:
Is the method not completely redundant now since it's just ´bucket->getId();´ or would you say it's cleaner if there is a specific ´getFeatureName()´ method that simply forwards the bucket ID?

Reply:
Yes, that's what i meant. `getFeatureName` is no longer necessary.